schema/table.go

```diff
@@ -26,7 +26,7 @@ type Transform func(table *Table) error
 
 type Tables []*Table
 
-// This is deprecated
+// Deprecated: SyncSummary is deprecated.
```

Comment:
This and some other changes here were just some comment clean-up in the file to better conform to Go standards